pandas/tests/groupby/test_groupby.py

```diff
@@ -2419,3 +2419,22 @@ def test_rolling_wrong_param_min_period():
     result_error_msg = r"__init__\(\) got an unexpected keyword argument 'min_period'"
     with pytest.raises(TypeError, match=result_error_msg):
         test_df.groupby("name")["val"].rolling(window=2, min_period=1).sum()
+
+
+@pytest.mark.parametrize("method", ["first", "last", "nth"])
+@pytest.mark.parametrize("nans", [None, np.nan])
+def test_groupby_last_first_nth_with_none(method, nans):
+    # GH29645
+    expected = Series([True])
+    data = (
+        Series([nans, "x", nans, "y", nans], index=[0, 0, 0, 0, 0])
+        .isna()
```

Comment:
I don't think isna is necessary.

Does parametrization over all missing values work? pd.NA, NaT etc

Reply:
If you mean something like `@pytest.mark.parametrize("nans", [None, np.nan, pd.NA, pd.NaT])` without `isna` then no, all tests fail. They all pass if `isna` is there

Reply:
If you check the same as in https://github.com/pandas-dev/pandas/issues/29645#issuecomment-885392862? We do not want to check for True False rather than for "y".

We have a missing value fixture, but yes I meant like this

Reply:
That's reasonable, I misunderstood